components/aws/sagemaker/tests/integration_tests/resources/config/simple-mnist-training/config.yaml

```diff
@@ -1,6 +1,7 @@
 PipelineDefinition: resources/definition/training_pipeline.py
 TestName: simple-mnist-training
 Timeout: 3600
+ExpectedTrainingImage: ((KMEANS_REGISTRY)).dkr.ecr.((REGION)).amazonaws.com/kmeans:1
 Arguments:
   region: ((REGION))
   image: ((KMEANS_REGISTRY)).dkr.ecr.((REGION)).amazonaws.com/kmeans:1
```

Comment:
Rather than specifying image for 1p algorithms, we can specify `algorithm_name: kmeans` and the code will find the image automatically. This also adds coverage for this functionality.

Reply:
Did that for HPO, I think this is also adding to coverage and we can have additional test for what you mentioned